src/ssl.c

```diff
@@ -55520,35 +55571,40 @@ int wolfSSL_CONF_cmd(WOLFSSL_CONF_CTX* cctx, const char* cmd, const char* value)
     (void)value;
     return WOLFSSL_FAILURE;
 }
+#endif /* !NO_WOLFSSL_STUB */
 
+#if defined(HAVE_EX_DATA) || defined(FORTRESS)
 /**
- * returns a new index or -1 on failure
+ * Issues unique index for the class specified by class_index.
+ * Other parameter except class_index are ignored.
+ * Currentry, following class_index are accepted:
```

Comment:
Spelling - "Currentry" should be "Currently"

Reply:
Fixed the spelling.